src/resources/views/crud/fields/select2_from_ajax.blade.php

```diff
@@ -93,6 +93,27 @@
 <!-- include field specific select2 js-->
 @push('crud_fields_scripts')
 <script>
+    var select2AjaxFetchSelectedEntry = function (element) {
```

Comment:
let's move this var inside `bpFieldInitSelect2FromAjaxElement()` please, right before we actually call it; there's no point in having it in the global namespace - it'll get overwritten pretty fast;